### PR TITLE
Alexa add reliabiity

### PR DESCRIPTION
**Pull request by JasBeilin**
## Status
- [x] Ready

## Related Issues
related: https://github.com/demisto/etc/issues/29347

## Description
Added a reliability parameter to integration

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [x] 5.0.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details: added required parameter

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by JasBeilin** [comment]
No test were added to the structure of code.
image

File: Packs/Alexa/Integrations/Alexa/Alexa.py [hunk 1]
```diff
@@ -106,11 +102,25 @@ def test_module_command():
 
 """EXECUTION BLOCK"""
 try:
+    params = demisto.params()
+    args = {
+        'threshold': int(params.get('threshold', 2000000)),
```
Comment (bakatzir) [inline Packs/Alexa/Integrations/Alexa/Alexa.py:107]: where the `2000000` came from?

File: Packs/Alexa/Integrations/Alexa/Alexa.py [hunk 2]
```diff
@@ -36,23 +30,24 @@ def alexa_fallback_command(domain):
     return formatted_result
 
 
-def alexa_domain_command():
+def alexa_domain_command(use_ssl, proxies, threshold, benign, reliability):
     domain = demisto.args().get('domain')
```
Comment (bakatzir) [inline Packs/Alexa/Integrations/Alexa/Alexa.py:34]: so you have to sets of args now, this is a bit confusing.

File: Packs/Alexa/Integrations/Alexa/Alexa.py (hunk 1 above)
Comment (JasBeilin) [reply Packs/Alexa/Integrations/Alexa/Alexa.py:107]: came from the YML.

File: Packs/Alexa/Integrations/Alexa/Alexa.py [hunk 3]
```diff
@@ -106,11 +101,27 @@ def test_module_command():
 
 """EXECUTION BLOCK"""
 try:
+    params = demisto.params()
+    domain = demisto.args().get('domain')
```
Comment (bakatzir) [inline Packs/Alexa/Integrations/Alexa/Alexa.py:105]: let's put this line in `demisto.command() == 'domain'` as we do not have domain when running the test module

Comment (JasBeilin) [reply Packs/Alexa/Integrations/Alexa/Alexa.py]: good catch, thanks!

**Review by bakatzir — APPROVED** [review]

**Review by bakatzir — APPROVED** [review]

**Pull request merged**